ios/Open In Thunder/Resources/content.js

```diff
@@ -0,0 +1,100 @@
+let instances = [
+    "aussie.zone",
```

Comment:
Is there any way to put this list in another file by itself? That would make it easier to update via the GitHub Actions.

Reply:
There might be a way, but this was the simplest way I found to do this - I tried to place it in a different file but it wouldn't import it. Not sure if this is a restriction or a setting that I would have to tweak to allow that to work!